ui/app/templates/vault/cluster/auth.hbs

```diff
@@ -101,17 +93,13 @@
           @onError={{fn (mut this.mfaErrors)}}
         />
       {{else}}
-        <AuthForm
```

Comment:
`<Mfa::MfaForm>` will also eventually live inside `Auth::Page` moving that during this PR is TBD